utils/parse.go

```diff
@@ -7,11 +7,12 @@ import (
 	"unicode"
 )
 
-func ParseShell(input string) []string {
+func ParseShell(input string) ([]string, error) {
 	// normalize quotation marks
 	r := strings.NewReplacer("‘", "'", "’", "'", "“", "\"", "”", "\"")
```

Comment:
since we are ignoring ', we probably don't need to "normalize" them in that first line of this function (slight perf boost, and possibly save us from unexpected behavior).

Reply:
As discussed, I'm keeping the normalization in so that the single quotation mark runes are consistent.